src/Microsoft.AspNetCore.SystemWebAdapters/Adapters/ForwardedHost.Shared.cs

```diff
@@ -7,14 +7,14 @@ namespace Microsoft.AspNetCore.SystemWebAdapters;
 
 internal readonly struct ForwardedHost
 {
-    public ForwardedHost(string host)
+    public ForwardedHost(string host, string? proto)
     {
         var idx = host.IndexOf(":", 0, StringComparison.Ordinal);
```

Comment:
This is broken if the host header is an IPv6 address.
```
::1
[::1]:80
```

See https://github.com/dotnet/aspnetcore/blob/0e255c5e3b86565a8d930f403de79bfc8b02cef5/src/Http/Http.Abstractions/src/HostString.cs#L160-L193

This could be addressed separately.

Reply:
Added an issue to track getting that fixed